CONTRIBUTING.md

```diff
@@ -24,50 +24,73 @@ python3 -c 'import secrets; print(secrets.token_hex(100))' > .credentials/SECRET
-2. For your GitHub user, in **Developer Settings**, generate a new [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+2.  For your GitHub user, in **Developer Settings**, generate a new [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
 
-   This is not strictly necessary just to run the service, but allows for more API calls and is therefore important for a production deployment.
+    This is not strictly necessary just to run the service, but allows for more API calls and is therefore important for a production deployment.
 
-   - Generate new token
-     - In **Resource owner** select the GitHub organisation associated with the application
-     - In **Repository access** select **Public Repositories (read-only)**
-     - In **Permissions**, set **Members** permissions to **Read-only**
-     - No other permissions are required
-   - Store the value in `.credentials/GH_TOKEN`
+    - Generate new token
+      - In **Resource owner** select the GitHub organisation associated with the application
+      - In **Repository access** select **Public Repositories (read-only)**
+      - In **Permissions**, set **Members** permissions to **Read-only**
+      - No other permissions are required
+    - Store the value in `.credentials/GH_TOKEN`
 
-3. In the GitHub organisation settings, [create an OAuth application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app):
+3.  In the GitHub organisation settings, [register a GitHub application](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app):
 
-   - In **Personal access tokens** approve the request under **Pending requests** if approval is required
-   - In **Developer settings** OAuth Apps, create a new application
+    - In **Personal access tokens** approve the request under **Pending requests** if approval is required
+    - In **Developer settings** GitHub Apps, create a new application
 
-     Store the **Client ID** in `.credentials/GH_CLIENT_ID`
+      - In **Repository Permissions** select **Administration (read-only)** and **(Metadata: read-only)**.
+      - In **Organization Permissions** select **Administration (read-only)** and **(Members: read-only)**.
 
-   - In the application settings **Generate a new client secret**
+      Store the **Client ID** in `.credentials/GH_CLIENT_ID`
 
-     Store the value in `.credentials/GH_SECRET`
+    - In the application settings / **General** / **Generate a new client secret**
+
+      Store the value in `.credentials/GH_SECRET`
+
+    - In the application settings / **General** / **Private keys** / **Generate a private key**
+
+      Store the value in `.credentials/GH_APP_PRIVATE_KEY`
+
+    - In the application settings / **Install App**
+
+      **Install** in the organisation account.
```

Comment:
This is missing the "grab the app installation ID" step

Reply:
Done